src/pages/patientView/clinicalInformation/CohortVariantCountCache.spec.ts

```diff
@@ -0,0 +1,273 @@
+import {default as CohortVariantCountCache, EntrezToKeywordList, VariantCountCacheType} from './CohortVariantCountCache';
+import { assert, default as chai } from 'chai';
+import sinon from 'sinon';
+import {VariantCount} from "../../../shared/api/CBioPortalAPIInternal";
+
+chai.config.truncateThreshold = 0;
+
+
+describe('CohortVariantCountCache', () => {
+
+    let data:VariantCount[];
+
+    let sampleIds:string[];
+
+    let fetchPromise:Promise<VariantCount[]>;
+
+    let initialCache:VariantCountCacheType = {
+        numberOfSamples: null,
+        mutationInGene: {},
+        keyword: {}
+    };
+
+    class TestCache extends CohortVariantCountCache {
```

Comment:
I think the problem I was running into was that the method is private so not accessible that way. That's the reason I did it this way.